backend/src/main/java/com/huellapositiva/domain/service/RemoteStorageService.java

```diff
@@ -65,4 +65,23 @@ private String getExtension(String fileName) {
         }
         return "";
     }
+
+    /**
+     * This method reads the bytes from the photo of a proposal and uploads it to the storage service
+     *
+     * @param photo
+     * @param volunteerId
+     * @return URL with the photo location in the storage
+     * @throws IOException
+     */
+    public URL uploadVolunteerPhoto(MultipartFile photo, String volunteerId) throws IOException {
+        String extension;
+        extension = getExtension(photo.getOriginalFilename());
+        if(!(".jpg".equalsIgnoreCase(extension)) || ".jpeg".equalsIgnoreCase(extension) || ".png".equalsIgnoreCase(extension) || ".gif".equalsIgnoreCase(extension)) {
+            throw new FileTypeNotSupportedException("photo file must be .jpg,.png,.jpeg,.gif");
+        }
```

Comment:
Extract valid extensions to a private set and look for contains

Reply:
It's done